sdk/spring/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
+* Fixed issues with how nested partition keys are used by `azure-spring-data-cosmos` - See [PR 38274](https://github.com/Azure/azure-sdk-for-java/pull/38274).
```

Comment:
Since this is an optimization, we should call it out in Other Changes section, something of this sort: 

"Optimized querying entities with nested partition keys by passing the nested partition key in `CosmosQueryRequestOptions`"

Reply:
We should also mention the id generation fix in case of bulk APIs (save all)